cmp/ag-grid/AgGrid.js

```diff
@@ -98,7 +98,7 @@ AgGrid.getRowHeightForSizingMode = (mode) => (XH.isMobileApp ? AgGrid.ROW_HEIGHT
 /**
  * Header heights (in pixels)
  */
-AgGrid.HEADER_HEIGHTS = {large: 36, standard: 32, compact: 28, tiny: 22};
+AgGrid.HEADER_HEIGHTS = {large: 28, standard: 24, compact: 20, tiny: 18};
```

Comment:
I like the look of large and standard, but compact and tiny feel too squished, especially if the column filter menu is enabled. I like how {compact: 22, and tiny: 20} look personally.
I also think the column filter menu icon top margin needs to be tweaked to 2px or 3px with this change: https://github.com/xh/hoist-react/blob/555a550a0b0534cb241770a87d90b07ee63c0b37/desktop/cmp/grid/impl/filter/ColumnHeaderFilter.scss#L12.